playbooks/foreman_proxy_content.yml

```diff
@@ -17,14 +17,14 @@
       foreman_installer_scenario: foreman-proxy-content
       foreman_installer_options_internal_use_only:
           - '--disable-system-checks
+            --certs-tar-file "{{ foreman_proxy_content_certs_tar }}"
             --foreman-proxy-content-enable-ostree true
             --foreman-proxy-trusted-hosts "{{ server_fqdn.stdout }}"
             --foreman-proxy-trusted-hosts "{{ ansible_nodename }}"
             --foreman-proxy-foreman-base-url "https://{{ server_fqdn.stdout }}"
             --foreman-proxy-register-in-foreman true
             --foreman-proxy-oauth-consumer-key "{{ oauth_consumer_key.stdout }}"
             --foreman-proxy-oauth-consumer-secret "{{ oauth_consumer_secret.stdout }}"
-            --foreman-proxy-content-certs-tar "{{ foreman_proxy_content_certs_tar }}"
```

Comment:
This will now fail for all versions except for nightly, I think we need a version check here to use this parameter in some cases.